tuner/tuner/libtuner_test.py

```diff
@@ -145,6 +147,24 @@ def test_parse_dispatch_benchmark_results():
         "2\tMean Time: 200.0\n",
         "Candidate 3 not completed",
     ]
+    expected_json_output = {
+        "context": {
+            "date": ANY,
+            "num_cpus": ANY,
+            "mhz_per_cpu": ANY,
+            "cpu_scaling_enabled": ANY,
```

Comment:
Should I add a comment explaining that these fields are dynamically generated, therefore they are ignored using `ANY`?